src/main/java/com/exasol/releasedroid/adapter/github/GitHubReleaseMaker.java

```diff
@@ -48,11 +49,19 @@ private String createGitHubRelease(final Repository repository) {
 
     private GitHubRelease createReleaseModel(final Repository repository, final String version) {
         final ReleaseLetter releaseLetter = repository.getReleaseLetter(version);
+        Optional<String> header = releaseLetter.getHeader();
+        if (header.isEmpty()) {
+            throw new IllegalStateException("Release header must not be empty.");
```

Comment:
Do we already use Error Codes in this project?

Reply:
yes, I will allocate a new error code